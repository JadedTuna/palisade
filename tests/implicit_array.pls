in {
	high secret: bool;
}
out {
	low arr[3]: bool;
}

x := 10;
y := false;
if (secret) {
	y = true;
}

arr[x] = y;
