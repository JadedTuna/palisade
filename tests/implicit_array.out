[1;35msecurity error: [0mcan't index with a high value in a low array
   1 | low arr[3] = [false, false, false];
   2 | high secret = 2;
   3 | arr[[1;35msecret[0m] = true;
       [1;35m~~~~[0m[1;35m^^^^^^[0m
